src/main/java/robot/subsystems/wrist/Wrist.java

```diff
@@ -0,0 +1,84 @@
+package robot.subsystems.wrist;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import robot.subsystems.wrist.commands.MoveToCube;
+
+import static robot.Constants.Wrist.*;
+import static robot.Ports.Wrist.MASTER;
+import static robot.Ports.Wrist.SLAVE;
+
+public class Wrist extends Subsystem {
+    private TalonSRX masterMotor = new TalonSRX(MASTER);
+    private VictorSPX slaveMotor = new VictorSPX(SLAVE);
+
+    public Wrist() {
+        masterMotor.setInverted(MASTER_INVERTED);
+        slaveMotor.setInverted(SLAVE_INVERTED);
+        masterMotor.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Absolute, PID_SLOT, TIMEOUT_MS);
+        masterMotor.config_kP(PID_SLOT, KP, TIMEOUT_MS);
+        masterMotor.config_kI(PID_SLOT, KI, TIMEOUT_MS);
+        masterMotor.config_kD(PID_SLOT, KD, TIMEOUT_MS);
+        slaveMotor.follow(masterMotor);
+    }
+
+    @Override
+    protected void initDefaultCommand() {
+        setDefaultCommand(new MoveToCube(5));
+    }
+
+    public void turnTo(double angle) {
+        angle %= 360;
+        masterMotor.set(ControlMode.MotionMagic, convertDegreesToTicks(angle) + applyForceByAngle(angle));
+    }
+
+    public void turnTo(State state) {
+        masterMotor.set(ControlMode.PercentOutput, state.applyForces());
+    }
+
+    public double convertTicksToDegrees(double ticks) {
+        return ticks / TICKS_PER_DEGREE;
+    }
+
+    public double convertDegreesToTicks(double degrees) {
+        return degrees * TICKS_PER_DEGREE;
+    }
+
+    public double getAngle() {
+        return convertTicksToDegrees(masterMotor.getSelectedSensorPosition());
+    }
+
+    public double applyForceByAngle(double angle) {
+        return convertDegreesToTicks(Math.cos(angle));
+    }

```

Comment:
In addition to the comment on the motion magic, this should be changed to give an arbitrary feedforward (power output) value, -1 to 1, an added "percent output" value.
The current code just adds distance for the motion magic to go and doesn't do anything to negate the force of gravity